src/pages/Profile/Profile.tsx

```diff
@@ -0,0 +1,206 @@
+                  {/* <FormHelperText id="component-error-text">Error</FormHelperText> */}
+                </FormControl>
+                <FormControl className={classes.formControl}>
+                  <InputLabel htmlFor="component-simple">Location</InputLabel>
+                  <Input
+                    id="component-simple"
+                    value={values.location}
+                    name="location"
+                    onChange={handleInputChange}
+                  />
+                  {/* <FormHelperText id="component-error-text">Error</FormHelperText> */}
+                </FormControl>
+                <FormControl className={classes.formControl}>
+                  <FormLabel component="legend">
+                    Do you use milk with coffee?
+                  </FormLabel>
+                  <RadioGroup
+                    className={classes.formRadio}
+                    value={values.milk}
+                    name="milk"
+                    onChange={handleInputChange}
+                  >
+                    <FormControlLabel
+                      value={true}
+                      control={<Radio />}
+                      label="Yes"
+                    />
+                    <FormControlLabel
+                      value={false}
+                      control={<Radio />}
+                      label="No"
+                    />
+                  </RadioGroup>
+                </FormControl>
+                <div>
+                  <FormControl className={classes.formControl}>
+                    <TextField
+                      className={classes.time}
+                      id="time"
+                      label="Start of work day"
+                      type="time"
+                      defaultValue="09:00"
+                      value={values.start}
+                      name="start"
+                      onChange={handleInputChange}
+                      InputLabelProps={{
+                        shrink: true
+                      }}
+                      inputProps={{
+                        step: 60 * 15
+                      }}
+                    />
+                  </FormControl>
+                  <FormControl className={classes.formControl}>
+                    <TextField
+                      className={classes.time}
+                      id="time"
+                      label="End of work day"
+                      type="time"
+                      defaultValue="18:00"
+                      name="end"
+                      value={values.end}
+                      InputLabelProps={{
+                        shrink: true
+                      }}
+                      inputProps={{
+                        step: 60 * 15
+                      }}
+                      onChange={handleInputChange}
+                    />
+                  </FormControl>
+                </div>
+              </form>
+            </Paper>
+          </Grid>
+        </Grid>
+      </main>
```

Comment:
can we create separate component for this? hard to read code. maybe we can reduce this with 2 mappers? what do you think?